index.d.ts

```diff
@@ -164,9 +164,9 @@ export interface Options extends RequestInit {
 	/**
 	Shortcut for sending JSON. Use this instead of the `body` option.
 
-	Accepts a plain object which will be `JSON.stringify()`'d and the correct header will be set for you.
+	Accepts any plain object or value, which will be `JSON.stringify()`'d and sent in the body with the correct header set.
 	*/
-	json?: unknown;
+	json?: any;
```

Comment:
This should be correctly typed. The keys are strings and the values can be: string, number, boolean, null, object, array.

Reply:
I'm not sure what you mean by "the keys are strings".  I've updated the type to explicitly list those types, and included `undefined`, which is technically allowed (though it serializes to `null`...)

Reply:
For TypeScript purposes, I think it should probably stay as `unknown`.

https://stackoverflow.com/questions/51439843/unknown-vs-any/51439876